suites/focal.toml

```diff
@@ -16,10 +16,11 @@ version = "1.67.2-1652812855"
 
 [[direct]]
 name = "discord"
-version = "0.0.89"
+version = "0.0.90-1"
 
     [[direct.urls]]
-    url = "https://dl.discordapp.net/apps/linux/${version}/discord-${version}.deb"
+#    url = "https://dl.discordapp.net/apps/linux/${version}/discord-${version}.deb"
+    url = "https://dl.discordapp.net/apps/linux/${version}/discord-0.0.89.deb"
```

Comment:
Should the first `${version}` also be manually set in that URL? Otherwise I get this:
![Screenshot_2025-04-15_10-37-28](https://github.com/user-attachments/assets/30e7e0c1-6e59-491d-8e29-eaaaa9678326)

Reply:
Yes